utils/filterPAS/README.md

```diff
@@ -0,0 +1,5 @@
+# filterPAS - module and scripts to filter & reannotate PAS BED files for relative quantification
```

Comment:
Need to document the workflow, inputs & outputs, how to call the function/script etc.